### PR TITLE
Updated Catalan strings

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: config/locales/ca.yml [hunk 1]
```diff
@@ -437,11 +539,81 @@ ca:
...
+      <h3 id="changes">Canvis a la nostra política de privacitat</h3>
+
+      <p>Si decidim canviar la nostra política de privadesa, publicarem aquests canvis en aquesta pàgina.</p>
+
+      <p>Aquest document és CC-BY-SA. Es va actualitzar per última vegada el 31 de maig de 2013.</p>
+
+      <p>Originalment adaptat a la <a href=\"https://github.com/discourse/discourse\">política de privadesa del Discurs</a>.</p>
     title: "%{instance} Condicions del servei i política de privadesa"
+  themes:
+    default: Mastodon
```
Comment (lynlynlynx) [inline config/locales/ca.yml]: Unnecessary translation. See #5741

**Comment by lynlynlynx** [comment]
Conflicts are resolved!

File: config/locales/simple_form.ca.yml [hunk 2]
```diff
@@ -4,53 +4,64 @@ ca:
     hints:
       defaults:
         avatar: PNG, GIF o JPG. Màxim 2MB. Serà escalat a 120x120px
+        digest: S'envia després d'un llarg període d'inactivitat amb un resum de les mencions que has rebut en la teva absència
         display_name:
-          one: Queda <span class="name-counter">1</span> caràcter
-          other: Queden <span class="name-counter">%{count}</span> caràcters
+          one: <span class="name-counter">1</span> character left
+          other: <span class="name-counter">%{count}</span> characters left
```
Comment (lynlynlynx) [inline config/locales/simple_form.ca.yml]: Translation reverted by merging. Is it okay @salvadorpla ?

File: config/locales/ca.yml [hunk 3]
```diff
@@ -130,23 +188,35 @@ ca:
...
-      undo: Desfés
+      undo: Desfer
+    email_domain_blocks:
+      add_new: Afegir nou
+      created_msg: S'ha creat el bloc de domini de correu electrònic
+      delete: Suprimeix
+      destroyed_msg: S'ha eliminat correctament el bloc del domini de correu
+      domain: Domini
+      new:
+      create: Crear bloc
```
Comment (lynlynlynx) [inline config/locales/ca.yml]: Need one more indent for `create: Crear bloc` .

**Review by lynlynlynx — DISMISSED** [review]

File: config/locales/simple_form.ca.yml (hunk 2 above)
Comment (salvadorpla) [reply config/locales/simple_form.ca.yml]: Yes, it's okay. Had some issues with git.

**Comment by lynlynlynx** [comment]
There are some mistakes of indent in simple_form.ca.yml, line19 and 23.
Probably, it'll be all done with this.
Sorry for the request being split.

**Comment by salvadorpla** [comment]
The Travis CI build claims 
"The following data requires normalization:
config/locales/ca.yml
Run `i18n-tasks normalize` to fix"
Must be done by me locally ?

File: app/javascript/mastodon/locales/ca.json [hunk 4]
```diff
@@ -187,7 +188,6 @@
   "status.load_more": "Carrega més",
   "status.media_hidden": "Multimèdia amagat",
   "status.mention": "Esmentar @{name}",
-  "status.more": "Més",
```
Comment (ykzts) [inline app/javascript/mastodon/locales/ca.json]: why delete?

File: app/javascript/mastodon/locales/ca.json [hunk 5]
```diff
@@ -85,6 +85,7 @@
   "empty_column.community": "La línia de temps local és buida. Escriu alguna cosa públicament per fer rodar la pilota!",
   "empty_column.hashtag": "Encara no hi ha res amb aquesta etiqueta.",
   "empty_column.home": "Encara no segueixes ningú. Visita {public} o fes cerca per començar i conèixer altres usuaris.",
+  "empty_column.home.inactivity": "La línia de temps Inici és buida. Si has estat inactiu durant un temps, es regenerarà aviat..",
```
Comment (ykzts) [inline app/javascript/mastodon/locales/ca.json]: This message is not used.

File: config/locales/simple_form.ca.yml [hunk 6]
```diff
@@ -4,53 +4,64 @@ ca:
...
         header: PNG, GIF o JPG. Màxim 2MB. Serà escalat a 700x335px
-        locked: Cal que aprovis manualment els seguidors i les publicacions es mostraran només als teus seguidors
+        locked: Requereix que aprovis manualment seguidors i les publicacions seran mostrades només als teus seguidors
         note:
-          one: Queda <span class="note-counter">1</span> caràcgter
-          other: Queden <span class="note-counter">%{count}</span> caràcters
+          one: <span class="note-counter">1</span> character left
+          other: <span class="note-counter">%{count}</span> characters left
```
Comment (ykzts) [inline config/locales/simple_form.ca.yml]: Is this English?

**Review by ykzts — CHANGES_REQUESTED** [review]

**Comment by ykzts** [comment]
cc @JeanGauthier

**Review by ykzts — APPROVED** [review]

**Pull request merged**